src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -191,6 +191,10 @@
     <Compile Include="..\Shared\LanguageParser\VisualBasictokenizer.cs">
       <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
     </Compile>
+    <Compile Include="..\Shared\PlatformNegotiation.cs">
```

Comment:
Do we have to compile this twice? Can we instead call the method from `Microsoft.Build.dll`?

Reply:
Ah, I see Roman asked the same question: https://github.com/dotnet/msbuild/pull/7480/files#r860829258

Reply:
Spoke with Roman offline and it wouldn't be worth it to move it to the microsoft.build.framework project as it relies on too many dependencies